wp-content/themes/reconasia/assets/_scss/components/_post-meta.scss

```
@@ -1,52 +1,68 @@
 @use 'abstracts' as *;
 
 .post-meta {
-  color: var(--post-meta);
-
   &__label {
     font-style: italic;
   }
 
   &__top {
     display: flex;
-    // @extend %font-label-lg;
   }
 
-  &__issue .post-meta__value,
   &__date {
-    letter-spacing: 1px;
-    text-transform: uppercase;
+    color: $color-black-165;
+    @extend %text-style-label-small-strong-caps;
   }
 
-  &__issue {
-    .post-meta__label {
-      text-transform: lowercase;
-    }
+  &__categories {
+    color: $color-link-primary-200;
+    @extend %text-style-label-x-small-strong-caps;
 
-    .post-meta__value {
-      font-weight: 600;
+    a:not(:last-of-type) {  
+      &::after {
+        content: '';
+        margin-right: rem(10);
+        margin-left: rem(10);
+        border-right: 1px solid $color-border-dark-130;
+      }
     }
-  }
 
-  &__date:not(:only-child) {
-    margin-left: rem(8);
+    > a {
```

```suggestion
    a {
```

You don't need the direct child selector here.